docs/endpoints/Legacy/README.md

```diff
@@ -0,0 +1,82 @@
+# Rogue API
+This is the Rogue API, it is used to capture activity from members.
+
+Most recent stable release is V3
+
+### Documentation
+[API v3](/docs/endpoints/v3/README.md)
+
+<!-- ## Endpoints
```

Comment:
Should all these endpoints be commented out? 👀

Reply:
you're right they shouldn't be!